autonomy/analyse/service.py

```diff
@@ -21,29 +21,168 @@
+                    },
+                    "additionalProperties": False,
+                    "minProperties": 1,
+                }
+            },
+            "required": ["ledger_apis"],
+        },
+    },
+    "required": ["config"],
+}
+
+ABCI_SKILL_SCHEMA = {
+    "type": "object",
+    "properties": {
+        "models": {
+            "type": "object",
+            "properties": {
+                "params": {
+                    "type": "object",
+                    "properties": {
+                        "args": {
+                            "type": "object",
+                            "properties": {
+                                "setup": {
+                                    "type": "object",
+                                    "required": [
+                                        "safe_contract_address",
+                                        "all_participants",
+                                    ],
+                                }
+                            },
+                            "required": [
+                                "setup",
+                                "tendermint_url",
+                                "tendermint_com_url",
+                                "tendermint_p2p_url",
+                                "service_registry_address",
+                                "share_tm_config_on_startup",
+                                "on_chain_service_id",
+                            ],
+                        },
+                    },
+                    "required": [
+                        "args",
+                    ],
+                },
+            },
+            "required": [
+                "params",
+            ],
+        }
+    },
+    "required": ["models"],
+}
 
-REQUIRED_PARAM_VALUES = (
-    "share_tm_config_on_startup",
-    "on_chain_service_id",
+ABCI_SKILL_MODEL_PARAMS_SCHEMA = {
+    "type": "object",
+    "properties": {
+        "setup": {
+            "type": "object",
+            "required": [
+                "safe_contract_address",
+                "all_participants",
+            ],
+        }
+    },
+    "required": [
+        "genesis_config",
+        "service_id",
+        "tendermint_url",
+        "max_healthcheck",
+        "round_timeout_seconds",
+        "sleep_time",
+        "retry_timeout",
+        "retry_attempts",
+        "keeper_timeout",
+        "observation_interval",
+        "drand_public_key",
+        "tendermint_com_url",
+        "tendermint_max_retries",
+        "tendermint_check_sleep_delay",
+        "reset_tendermint_after",
+        "consensus",
+        "cleanup_history_depth",
+        "cleanup_history_depth_current",
+        "request_timeout",
+        "request_retry_delay",
+        "tx_timeout",
+        "max_attempts",
+        "service_registry_address",
+        "on_chain_service_id",
+        "share_tm_config_on_startup",
+        "tendermint_p2p_url",
+        "setup",
+    ],
+}
```

Comment:
@DavidMinarsch make sure we have every required model parameters in the `skill.yaml`